frontend/src/pages/auth/login.tsx

```diff
@@ -214,20 +214,33 @@ export const LoginPage: FC = () => {
 
     const handleHostNameChange = useCallback((newHostName: string) => {
         if (databaseType.id !== DatabaseType.MongoDb || !newHostName.startsWith("mongodb+srv://")) {
-            return setHostName(newHostName);
-        }
-        const url = new URL(newHostName);
-        setHostName(url.hostname);
-        setUsername(url.username);
-        setPassword(url.password);
-        setDatabase(url.pathname.substring(1));
-        const advancedForm = { "Port": "27017", "URL Params": `?${url.searchParams.toString()}`, "DNS Enabled": "false" };
-        if (url.port.length === 0) {
-            advancedForm["Port"] = "";
-            advancedForm["DNS Enabled"] = "true";
+            if (databaseType.id === DatabaseType.Postgres && newHostName.startsWith("postgres://")) {
```

Comment:
Here could you also check if it starts with "postgresql://" as that's also a valid start to the connection string